wayfinder-cloudaccess.tf

```diff
@@ -37,6 +38,18 @@ resource "kubectl_manifest" "wayfinder_azure_cloudinfo_cloudaccessconfig" {
   })
 }
 
+resource "kubectl_manifest" "wayfinder_azure_privatelinkmanager_cloudaccessconfig" {
+  count      = var.enable_k8s_resources && var.enable_wf_cloudaccess && var.enable_wf_privatelinks ? 1 : 0
+  depends_on = [time_sleep.after_kubectl_manifest_cloud_identity]
+
+  yaml_body = templatefile("${path.module}/manifests/wayfinder-azure-privatelinkmanager-cloudaccessconfig.yml.tpl", {
+    region          = var.location
+    subscription_id = data.azurerm_subscription.current.subscription_id
+    tenant_id       = data.azurerm_subscription.current.tenant_id
+    identity        = "cloudidentity-azure"
```

Comment:
small improvement: make this a local and pass to the three or so places that it's referenced.